pywhy_graphs/algorithms/pag.py

```diff
@@ -551,6 +552,7 @@ def pds(
     ----------
     .. footbibliography::
     """
+    print("HAHAHAHAHHA")
```

Comment:
I assume this is just a debugging tool but should be removed before committing

Reply:
Sure!